hr_holidays_leave_auto_approve/models/hr_leave.py

```diff
@@ -27,12 +27,14 @@ def _apply_auto_approve_policy(self):
     @api.model
     def create(self, values):
         auto_approve = self._get_auto_approve_on_creation(values)
-        tracking_disable = self.env.context.get('tracking_disable', False)
-        tracking_disable = tracking_disable or auto_approve
-        res = super(
-            HrLeave, self.with_context(
-                tracking_disable=tracking_disable)
-            ).create(values)
+        tracking_disable = self.env.context.get('tracking_disable')
+        mail_skip = self.env.context.get('mail_activity_automation_skip')
```

Comment:
similar here: default value is no needed?

Reply:
Same as above